src/Orchard.Hosting/DefaultOrchardHost.cs

```diff
@@ -201,14 +203,14 @@ public ShellContext CreateShellContext(ShellSettings settings)
         /// <summary>
         /// Creates a transient shell for the default tenant's setup.
         /// </summary>
-        private ShellContext CreateSetupContext()
+        private async Task<ShellContext> CreateSetupContextAsync()
```

Comment:
why? there is an async call under it, so why shouldnt I turn this to async and push further up?